single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,365 @@
+
+- Step 2, Set up the Copy Data Activity by configuring the Source and Destination tab. First, switch to the Source tab, then select 'external' for the Data Storage Type. Subsequently, for the connection, create a new HTTP link. On the creation page, enter the server URL as below. Finally, click the 'Create' button to complete the setup.
+  
+- The URL is: <https://stmdwpublic.blob.core.windows.net/datasets/nyc-yellow-tripdata-2022>
+
+  ![data_pipeline_04](./images/data_pipeline/data_pipeline_04.png)
+
+- Step 3, Create parameters for our pipeline. We need to return to the pipeline design panel and click anywhere on the blank area. Then, select 'Parameter' at the place shown in the image, and add the following three parameters:
+  
+  ![data_pipeline_05](./images/data_pipeline/data_pipeline_05.png)
+  
+  - __urls__, Type: Array, Default Value:
+
+    ``` json
+    ["/yellow_tripdata_2022-01.parquet",
+    "/yellow_tripdata_2022-02.parquet",
+    "/yellow_tripdata_2022-03.parquet",
+    "/yellow_tripdata_2022-04.parquet",
+    "/yellow_tripdata_2022-05.parquet",
+    "/yellow_tripdata_2022-06.parquet",
+    "/yellow_tripdata_2022-07.parquet",
+    "/yellow_tripdata_2022-08.parquet",
+    "/yellow_tripdata_2022-09.parquet",
+    "/yellow_tripdata_2022-10.parquet",
+    "/yellow_tripdata_2022-11.parquet",
+    "/yellow_tripdata_2022-12.parquet",
+    "/taxi_zone_lookup.csv"]
+    ```
+
+  - __landing_path__, Type: String; Default Value: `01_landing`.
+  - __client_secret__, Type: SecureString, Default Value: left blank.
+
+- Step 4, Return to the Settings tab of the ForEach Activity, set the Batch count value to 20, this limits the maximum number of files to be downloaded concurrently. Then, in the Items setting, select the parameter 'URLs' that we just created, or directly enter '@pipeline().parameters.URLs'. Now we completed the configuration of the ForEach Activity.
+  
+  ![data_pipeline_06](./images/data_pipeline/data_pipeline_06.png)
+
+- Step 5, Return to the Source tab of the Copy data Activity within the ForEach Activity. In the connection, select the HTTP link that we just created. The Connection type should be HTTP. Then, enter '@item()' in the Relative URL, and select Binary as the File format. This completes the configuration of the Source in the Copy data Activity.
+  
+  ![data_pipeline_07](./images/data_pipeline/data_pipeline_07.png)
+
+- Step 6, Switch to the Destination tab and make the following configurations as shown in the image. Please note, if you have not created a Lakehouse, you need to create a new Lakehouse named as '[prefix]_lh' in the workspace first, then select this Lakehouse here. If you have already created a Lakehouse, then you can directly select the Lakehouse you have created here. The file path here is the second parameter 'landing_path' that we just created, so our Copy data Activity will download the data to the corresponding folder in our Lakehouse.
+  - File Path: __@pipeline().parameters.landing_path__
```

Comment:
The File Path instructs user to use "landing_path" but the image shows "SavePath" which is confusing.

Reply:
I changed this before. It seems that I forgot to update this picture. Thank you for pointing out that. and just has been updated.